deepchem/models/torch_models/chemberta.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+from deepchem.models.torch_models.hf_models import HuggingFaceModel
+from transformers.models.roberta.modeling_roberta import (
+    RobertaConfig, RobertaForMaskedLM, RobertaForSequenceClassification)
+from transformers.models.roberta.tokenization_roberta_fast import \
+    RobertaTokenizerFast
+
+
+class Chemberta(HuggingFaceModel):
+
+    def __init__(self,
+                 task: str,
+                 mode: Optional[str] = None,
+                 tokenizer_path: str = 'seyonec/PubChem10M_SMILES_BPE_60k',
+                 **kwargs):
+        tokenizer = RobertaTokenizerFast.from_pretrained(tokenizer_path)
+        if task == 'pretraining':
```

Comment:
There should be two pretraining tasks, mlm-pretraining, mtr-pretraining following chemberta-2 paper

Reply:
Documenting offline discussion, we should follow same convention @tonydavis629 is following in the GCNModular for now, but right fix is to implement the Task API in subsequent PRs

Reply:
If I am not wrong, GNNModular uses an attribute named `task`to denote the type of unsupervised task. Am I correct here @tonydavis629 ? 

I will go ahead and use the same.

Reply:
added task here as well as updated in HuggingFaceModel base class in #3389

Reply:
makes sense, I have updated to remove the use of `mode` and to use only `task`.